librocksdb-sys/src/lib.rs

```diff
@@ -11,1024 +11,11 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-//
-
-//! Raw bindings for RocksDB.
-//!
-//! This is simply a raw interface to the RocksDB C API. It is intended to underpin a higher-level library rather than for direct use.
 
-#![allow(dead_code, non_camel_case_types, non_upper_case_globals, non_snake_case)]
+#![allow(non_camel_case_types)]
+#![allow(non_snake_case)]
+#![allow(non_upper_case_globals)]
```

Comment:
I think adding `#![allow(dead_code)]` here may address the current build error, as it's generating >4mb of unused variable warnings, causing travis to abort.

Reply:
actually, this is being hit due to warnings emitted during the C++ compilation, and the issue is also happining on master right now, so I think the CI environment has changed. I'll try to address this on master and I'll re-trigger these tests!

Reply:
Awesome, thanks @spacejam!